src/main/java/org/folio/service/manager/input/InputDataManagerImpl.java

```diff
@@ -70,9 +71,9 @@ public InputDataManagerImpl(Context context) {
   }
 
   @Override
-  public void init(JsonObject request, Map<String, String> params) {
+  public void init(JsonObject request, JsonObject mProfile, Map<String, String> params) {
```

Comment:
Please use whole name "mappingProfile" instead of "mProfile"

Reply:
sure, will rename it